Project.toml

```
@@ -3,6 +3,9 @@ uuid = "cb0561c3-42a1-46a3-8749-4ae46de0afcc"
 authors = ["Invenia Technical Computing Corporation"]
 version = "0.1.0"
```

```suggestion
version = "1.0.0"
```

this has been stable internally for a while. No reason not to go for 1.0